contracts/src/v0.8/dev/interfaces/OCR2DRRegistryInterface.sol

```diff
@@ -0,0 +1,156 @@
+   * @param callbackGasLimit - How much gas you'd like to receive in your
+   * fulfillRequest callback. Note that gasleft() inside fulfillRequest
+   * may be slightly less than this amount because of gas used calling the function
+   * (argument decoding etc.), so you may need to request slightly more than you expect
+   * to have inside fulfillRequest. The acceptable range is
+   * [0, maxGasLimit]
+   * @param data Encoded OCR2DR request data, use OCR2DRClient API to encode a request
+   * @return requestId - A unique identifier of the request. Can be used to match
+   * a request to a response in fulfillRequest.
+   */
+  function sendRequest(
+    bytes32 keyHash,
+    uint64 subId,
+    uint16 minimumRequestConfirmations,
+    uint32 callbackGasLimit,
+    bytes calldata data
+  ) external returns (bytes32);
+
+  /*
+   * @notice Conclude billing for an OCR2DR request
+   * @param proof contains the proof and response
+   * @param rc request commitment pre-image, committed to at request time
+   * @return payment amount billed to the subscription
+   * @dev simulated offchain to determine if sufficient balance is present to fulfill the request
+   */
+  function fulfillRequest(
+    bytes32 requestId,
+    bytes32 keyHash,
+    RequestCommitment memory rc,
+    bytes calldata response,
+    bytes calldata err
+  ) external returns (uint96);
+
+  /**
+   * @notice Create a new OCR2DR subscription.
+   * @return subId - A unique subscription id.
+   * @dev You can manage the consumer set dynamically with addConsumer/removeConsumer.
+   * @dev Note to fund the subscription, use transferAndCall. For example
+   * @dev  LINKTOKEN.transferAndCall(
+   * @dev    address(REGISTRY),
+   * @dev    amount,
+   * @dev    abi.encode(subId));
+   */
+  function createSubscription() external returns (uint64 subId);
+
+  /**
+   * @notice Get a OCR2DR subscription.
+   * @param subId - ID of the subscription
+   * @return balance - LINK balance of the subscription in juels.
+   * @return reqCount - number of requests for this subscription, determines fee tier.
+   * @return owner - owner of the subscription.
+   * @return consumers - list of consumer address which are able to use this subscription.
+   */
+  function getSubscription(uint64 subId)
+    external
+    view
+    returns (
+      uint96 balance,
+      uint64 reqCount,
+      address owner,
+      address[] memory consumers
+    );
+
+  /**
+   * @notice Request subscription owner transfer.
+   * @param subId - ID of the subscription
+   * @param newOwner - proposed new owner of the subscription
+   */
+  function requestSubscriptionOwnerTransfer(uint64 subId, address newOwner) external;
+
+  /**
+   * @notice Request subscription owner transfer.
+   * @param subId - ID of the subscription
+   * @dev will revert if original owner of subId has
+   * not requested that msg.sender become the new owner.
+   */
+  function acceptSubscriptionOwnerTransfer(uint64 subId) external;
```

Comment:
What is the reason for needing to request & accept a transfer? Why can't the subscription owner just transfer a subscription in a single step?